improver/wxcode/utilities.py

```diff
@@ -90,7 +135,7 @@ def weather_code_attributes() -> Dict[str, Any]:
     return attributes
 
 
-def expand_nested_lists(query: Dict, key: str) -> List:
+def expand_nested_lists(query: Dict[str, Any], key: str) -> List:
```

Comment:
Should this be as shown below to include the type of the output?
```suggestion
def expand_nested_lists(query: Dict[str, Any], key: str) -> List[Any]:
```